pom.xml

```diff
@@ -60,10 +60,10 @@
     </scm>
 
     <properties>
-        <spring.version>5.2.13.RELEASE</spring.version>
+        <spring.version>5.2.22.RELEASE</spring.version>
         <spring-boot.version>2.3.2.RELEASE</spring-boot.version>
```

Comment:
May be we should bump it too?

Reply:
@ArtDu because `spring-data-commons` in spring-boot sequence version numbers, take a look Compile Dependencies for `spring-data-commons`

Reply:
> @ArtDu because `spring-data-commons` in spring-boot sequence version numbers, take a look Compile Dependencies for `spring-data-commons`

I can't find something like spring-boot in there https://github.com/spring-projects/spring-data-commons/blob/main/pom.xml

Reply:
Also bumped to 2.6.3

Reply:
@ArtDu when I added this library, Sping Boot had different version numbers from the Sping framework. Have they unified it by this time?

Reply:
Noop, I'm not talking about version difference between spring boot and spring framework. I don't understand why we use variable `spring-boot-version` in spring-data-commons library